front/admin/cli.ts

```diff
@@ -56,6 +57,20 @@ const workspace = async (command: string, args: parseArgs.ParsedArgs) => {
           workspaceId: w.id,
         }),
       ]);
+      await Promise.all([
+        VaultResource.makeNew({
```

Comment:
`nit` we could have one function to create default vault resource on the resource itself as we should not need to duplicate this code in so many different places.